Apps/WebClient/src/ClientApp/app/components/navmenu/navHeader.vue

```diff
@@ -2,7 +2,7 @@
   <b-navbar toggleable="lg" type="dark">
     <!-- Brand -->
     <b-navbar-brand>
-      <router-link to="/">
+      <router-link  to="/timeLine">
```

Comment:
Did you run "npm run lintfix" before comiting?

Reply:
I did, was I not suppose to ?

Reply:
You should run it. There is an extra space before 'to' that makes me think that i has not be run yet.